argus/config.py

```diff
@@ -129,9 +129,37 @@ def _parse_environment(self, key):
             values[section][subkey] = opt_value
         values = values_factory(config_file, values)
 
-        start_commands = self._parser.getlist(key, 'start_commands')
-        stop_commands = self._parser.getlist(key, 'stop_commands')
-        return key, values, preparer, start_commands, stop_commands
+        start_commands = self._load_commands(key, 'start_commands')
+        stop_commands = self._load_commands(key, 'stop_commands')
+        list_services_commands = self._load_commands(
+            key, 'list_services_commands')
+        filter_services_regexes = self._load_commands(
+            key, 'filter_services_regexes')
+        start_service_command = self._load_commands(
+            key, 'start_service_command')
+        if start_service_command:
+            start_service_command = start_service_command[0]
+        stop_service_command = self._load_commands(
+            key, 'stop_service_command')
+        if stop_service_command:
```

Comment:
I don't want it to throw an error if the key is not found so I used _load_commands which handles the key not found case and returns None.